frontend/src/redux/actionTypes.js

```diff
@@ -13,3 +13,12 @@ export const AUTH_LOGOUT = "AUTH_LOGOUT";
 
 // CLEAR MESSAGES
 export const AUTH_CLEAR_MESSAGES = "AUTH_CLEAR_MESSAGES";
+
+// SEARCH
+export const SEARCH = "SEARCH";
+export const SEARCH_LOADING = "SEARCH_LOADING";
+export const SEARCH_COMPLETED = "SEARCH_COMPLETED";
+
+//PROFILE
+export const PROFILE_INFO = "PROFILE_INFO";
+export const PROFILE_INFO_LOADING = "PROFILE_INFO_LOADING";
```

Comment:
don't you need a profile info success action type here?

Reply:
profile info loading action type is added

Reply:
It also needs action and a reducer in order to use it when api call is succeeded